app/services/claims/fetch_eligible_advocate_categories.rb

```diff
@@ -10,7 +10,7 @@ def initialize(claim)
 
     def call
       return unless claim&.agfs?
-      return fee_reform_categories if using_fee_reform?
+      return fee_reform_categories if using_fee_reform? || Offence.in_scheme_ten.include?(claim.offence)
```

Comment:
👍 (although, it feels `Offence.in_scheme_ten.include?(claim.offence)` seems to be something that can also determine the claim's fee scheme?)